environment.yaml

```diff
@@ -17,6 +17,8 @@ nodes:
     box: centos/6
   centos-5:
     box: gutocarvalho/centos5x64nocm
+  ubuntu-18.04:
+    box: ubuntu/bionic64
```

Comment:
First Puppet 5 package available for Bionic is 5.5.3, so versions below should be avoided. Because the default is still 1.10.12, the combination is invalid.

Reply:
I'll install the puppet agent only when we have puppet 5